app/helpers/spree_stripe/checkout_helper_decorator.rb

```diff
@@ -0,0 +1,15 @@
+module SpreeStripe
+  module CheckoutHelperDecorator
+    def checkout_payment_sources(payment_method = nil)
```

Comment:
@mad-eel let's add a check if payment method is a stripe method, if not, just call super